test/unit/spec/EnrollChoices_spec.js

```diff
@@ -775,6 +776,16 @@ function (Okta, OktaAuth, Util, EnrollChoicesForm, Beacon, Expect, Router,
               expect(test.form.factorCardinalityText(factorName)).toBe('(3 set up)');
             });
         });
+        itp('checks correct order after enroll', function () {
+          return setupMultipleFactorEnrollments(
+            factorName, 'ACTIVE', 'OPTIONAL',{'enrolled': 1, 'minimum': 1, 'maximum': 3}
+          ).then(function (test) {
+            var factorRows = test.form.optionalFactorList();
+            var factorIndex = factorRows.find(`[data-se="${factorName}"]`).index();
+            var expectedIndex = FactorUtil.getFactorSortOrder(null, factorName === 'U2F' ? 'u2f' : 'token:software:totp') - 1;
```

Comment:
This still won't work when we implement cardinality for other factors. You could either pass the factorType to this function, or enhance `factorEnrollList` to map factorName to factorType also.